ex11/part5/src/index.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react";
+import ReactDOM from "react-dom";
+import Cb from "./colorblockbriten";
+import Cd from "./colorblockdarken";
+
+const App = () => {
+    const [color, setColor] = useState("red");
+
+    function colorChanged(e) {
+        const newValue = e.target.value;
+        setColor(newValue);
+    }
+    const blocks = [1,2,3,4,5,6,7,8,9,10]
+    const changeDelta = Math.floor(100/(blocks.length/2))
+    const dirChangeAt = Math.floor(blocks.length/2)
+    let change = 100
+    return (
+        <div>
+            <h1>ex 11 part 6</h1>
+            color: <input onInput={colorChanged} value={color}/>
+            {blocks.map((value, index) => {
```

Comment:
There is no need to break this into 2 components. You can use the same element, not even a component and only change the lighten/darken property. Or even better, use programmatic `background: hsl(...)`